client/src/test/kotlin/io/provenance/client/PbClientTest.kt

```diff
@@ -62,6 +66,27 @@ class PbClientTest {
         }
     }
 
+    @Test
+    @Ignore("requires grpc connection")
+    fun `Simulated BROADCAST_MODE_BLOCK works`() {
```

Comment:
Same with this test, I ran it against testnet locally and it works, but ignored as I don't have the scope to stand up an int test framework properly at the moment for this repo